src/stopwords.c

```diff
@@ -188,3 +189,28 @@ void ReplyWithStopWordsList(RedisModuleCtx *ctx, struct StopWordList *sl) {
   RedisModule_ReplySetArrayLength(ctx, i);
   TrieMapIterator_Free(it);
 }
+
+char **GetStopWordsList(struct StopWordList *sl) {
+  size_t size = sl->m->cardinality;
+  if (size == 0) {
+    return NULL;
+  }
+
+  char **list = rm_malloc((size + 1) * sizeof(list));
```

Comment:
If we create list on the fly why not return the size instead of make it NULL terminated?
also we should mention that the list should be free by the caller as well as the values in it.